api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/MedicationTranslatorImplTest.java

```diff
@@ -59,12 +62,14 @@
 	public void setup() {
 		drug = new Drug();
 		medicationTranslator = new MedicationTranslatorImpl();
-		medicationTranslator.setConceptService(conceptService);
 		medicationTranslator.setConceptTranslator(conceptTranslator);
 		
 		Concept drugConcept = new Concept();
 		drugConcept.setUuid(DRUG_CONCEPT_UUID);
 		drug.setConcept(drugConcept);
+		drug.setMaximumDailyDose(3.5);
```

Comment:
@jecihjoy I think that all these values should be stored in a variable to avoid any confusion in tests written by other contributors. Maybe you have done this intentionally